src/scraper.rs

```diff
@@ -123,26 +132,28 @@ impl Scraper {
             response::ResponseData::Other(data) => data,
         };
 
-        //Create a scope to unlock path_map automagicly
-        {
+        // Create a scope to unlock path_map automagicly
+        { 
             let path_map = scraper.path_map.lock().unwrap();
             let path = path_map.get(url.as_str()).unwrap();
-            match response.filename {
-                Some(filename) => {
-                    disk::save_file(&filename, &data, &scraper.args.output);
-
-                    disk::symlink(path, &filename, &scraper.args.output);
-                }
-                None => {
-                    disk::save_file(path, &data, &scraper.args.output);
+    
+            if !scraper.exclude.is_match(&url.as_str()) && scraper.include.is_match(url.as_str())  {
+                match response.filename {
+                    Some(filename) => {
+                        disk::save_file(&filename, &data, &scraper.args.output);
+                        disk::symlink(path, &filename, &scraper.args.output);
+                    }
+                    None => {
+                        disk::save_file(path, &data, &scraper.args.output);
+                    }
                 }
             }
         }
 
         scraper.visited_urls.lock().unwrap().insert(url.to_string());
 
         if scraper.args.verbose {
-            info!("Downloaded {}", url);
+            info!("Visited {}", url);
```

Comment:
Changed it to visited, as we are not always downloading now.

Reply:
```suggestion
            info!("Visited: {}", url);
```

Reply:
✔️